### PR TITLE
Correct sleeping MIC behaviour, correct button indentation error

### PR DESCRIPTION
**Pull request by infradom**
Just a suggestion for MIC, feel free to solve it differently (see @kars remarks)
Correct button indentation error

### CODE REVIEW

**Comment by wills106** [comment]
Do you think this will also help with [this](https://github.com/wills106/homeassistant-solax-modbus/issues/88#issuecomment-1207287934) or is that more likely because the USB device can't communicate with anything? So it ends up spamming the USB Port?

**Comment by wills106** [comment]
Sorry pressed wrong button 🤣

**Comment by infradom** [comment]
> Do you think this will also help with [this](https://github.com/wills106/homeassistant-solax-modbus/issues/88#issuecomment-1207287934) or is that more likely because the USB device can't communicate with anything? So it ends up spamming the USB Port?

I dont think it is related to the USB/serial ussue

File: custom_components/solax_modbus/__init__.py [hunk 1]
```diff
@@ -1345,32 +1316,40 @@ def read_modbus_input_registers_mic(self):
...
+            self.data["feedin_power"] = output_power
+
+            self.data["feedin_power_r"] = output_power_phase_r
+            self.data["feedin_power_s"] = output_power_phase_s
+            self.data["feedin_power_t"] = output_power_phase_t
+            self.data["pv_power_1"] = pv_power_1
+            self.data["pv_power_2"] = pv_power_2
+            self.data["pv_total_power"] = pv_power_1 + pv_power_2
+            self.data["total_yield"] = round(total_yield * 0.0001, 2)
+            self.data["today_yield"] = round(today_yield * 0.001, 2)
```
Comment (Kars-de-Jong) [inline custom_components/solax_modbus/__init__.py:1353]: This one keeps updating, even when the inverter is in run mode Waiting. I see it go to 0 at around 0:00.

Comment (infradom) [reply custom_components/solax_modbus/__init__.py:1353]: Thanks Kars, but can you be more precise? About which one are you talking? Only today_yield or also other variables? We disabled them all for the time being, and in the beta7  we event dont try to read them if run_mode is not "waiting"

Comment (Kars-de-Jong) [reply custom_components/solax_modbus/__init__.py:1353]: I indeed meant `today_yield`, but there are indeed more values that are also measured when the inverter is waiting, or are at least OK when showing 0, these are all of them:

- Today’s yield
- DC voltages
- DC and AC currents
- DC and AC powers

Which leaves measurements to be disabled when waiting:
- AC voltages
- AC frequency
- Total yield

I think I have them all now…

**Pull request merged**